data/initrd/scripts/url/hmc/menu

```diff
@@ -0,0 +1 @@
+HMC
```

Comment:
'Hardware Management Console' is even longer than 'Digital Video Disk', yet we use 'DVD'.

My impression was that HMC is a well-known term for s390 admins. But you're from the s390 side. If you feel more comfortable with 'Hardware Management Console', then so be it. 'Hdw. Mgmt. Console', however, looks really awkward to me.

> Also, is the lack of a line-end character here of any importance?

The file holds the string for the menu entry, no newline needed. So, why waste a byte just because github shows a red symbol?